ocdskingfisherprocess/cli/commands/new_transform_compiled_releases.py

```diff
@@ -0,0 +1,31 @@
+import ocdskingfisherprocess.database
+import ocdskingfisherprocess.cli.commands.base
+from ocdskingfisherprocess.transform.compile_releases import CompileReleaseTransform
+
+
+class CheckCLICommand(ocdskingfisherprocess.cli.commands.base.CLICommand):
+    command = 'new-transform-compiled-releases'
```

Comment:
I just thought that when we have multiple transforms, it would be good to be explicit what command creates a transform. Having the common prefix on things like "new-transform-convert-1-0-to-1-1" will help people understand what's going to happen as they use the system.

Reply:
Naming makes sense, then. I wonder whether the command should be `new-transform`, which takes a `transform_type` as an argument. (I guess it depends whether some transforms will take additional arguments – though in the current architecture I don't know where such arguments would be stored.)